assignment/scripts/loops-practice.js

```diff
@@ -40,8 +70,33 @@ console.log('---- 3. While loop ----');
 // 3.a. TODO: Write a while loop to console.log each star in the 'stars' array
 console.log('Some stars using while:');
 
+let star = stars.length;
```

Comment:
Opposite problem of the last comment. This isn't a start, it's a length! Good variable names are important for humans that have to read your code (including yourself when you come back to it later).

```suggestion
let starsLength = stars.length;
```

You can also just not create this variable. You can reference `stars.length` on Line 76 fine.